src/jbuild

```diff
@@ -46,10 +46,18 @@
      (action
         (bash "rm -rf ${SCOPE_ROOT}/../../src/.flambda_flags ${SCOPE_ROOT}/../../_bin"))))
 
-(executables
-    ((names (Record Infer Verify))
-     (public_names (lig-record lig-infer lig-verify))
+;;(executables
+;;    ((names (Record Infer Verify ))
+;;     (public_names (lig-record lig-infer lig-verify ))
+;;     (ocamlopt_flags (:standard (:include .flambda_flags)))
+;;     (libraries (core core_extended))
+;;     (preprocessor_deps ((file config.h)))
+;;     (preprocess (pps (ppx_optcomp)))))
```

Comment:
Please uncomment these.

If you don't need to build these targets, you could always write `jbuild build _build/default/src/RunPIE.exe`
Although it's worth looking into your build issues with these targets.

Reply:
Yes. I have fixed it. Turned out using multiple build targets (multiple stanzas of executable) is really awkward and doesn't seem to permit sharing of modules. So I added my List_Tests into the current list of executables.